### ISSUE TITLE
System Generated Html Component: WCAG 4.1.2: ARIA hidden element must not contain focusable elements (th[aria-label="Contact\ Roles\ Assigned"])

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#26]
**Issue**: `ARIA hidden element must not contain focusable elements` ([`aria-hidden-focus`](https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI))

**Target application**: [OCProfile · Custom Portal](https://mymatterssit.microsoftcrmportals.com/OCProfile/?id=f88eeb86-6c44-e811-a952-000d3a30dc0a)

**Element path**: th[aria-label="Contact\ Roles\ Assigned"]

**Snippet**:

    <th style="width:16.129032258064516%;" class="sort-disabled" tabindex="0" aria-label="Contact Roles Assigned" aria-hidden="true">Contact Roles Assigned</th>

**How to fix**:

    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM

**Environment**:
Chrome version 73.0.3683.103

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
